lib/Mojolicious/Renderer.pm

```diff
@@ -108,6 +108,12 @@ sub render {
   # Inheritance
   my $content = $stash->{'mojo.content'} //= {};
   local $content->{content} = $output =~ /\S/ ? $output : undef if $stash->{extends} || $stash->{layout};
+  if ($stash->{inline_layout}) {
+    @$options{qw(template inline)} = (undef, delete $stash->{inline_layout});
+    $options->{format} = $stash->{format} || $self->default_format;
```

Comment:
I don't think this line has any test coverage.

Reply:
I’ve removed that line as there is no way to specify the layout format. I suppose that we could introduce a new keyword for that, but it’s probably not worth it.